src/components/EventDetails/EventDetails.js

```diff
@@ -841,6 +843,32 @@ export class EventDetail extends React.Component<Props, State> {
           ],
         };
         break;
+      case SMART_WALLET_ACCOUNT_DEVICE_ADDED:
+        eventData = {
+          name: 'New Smart Wallet account device',
+          actionTitle: 'Added',
```

Comment:
This action might be pending, so it would be better to handle its title as we handle titles of collectibles' transactions.